pkg/controller/s3/bucket/taggingConfig_test.go

```diff
@@ -217,6 +217,91 @@ func TestTaggingObserve(t *testing.T) {
 	}
 }
 
+func TestCompareS3Tags(t *testing.T) {
+	observedTag1 := s3.Tag{
+		Key:   aws.String("tenant"),
```

Comment:
Please use more general test cases as we would merge this into upstream later.

Reply:
good point